components/modal/modal.css

```diff
@@ -0,0 +1,65 @@
+@import url('../../style.css');
+
+/* Modal Component */
+
+.modal-container {
+  background: rgba(0, 0, 0, 0.6);
+  display: none;
+  /* this means it will take the entire page */
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
+
+.show-modal {
```

Comment:
Suggestion: you can define a utility class for flex props and reuse in every components instead of repeating it

Reply:
This is being toggled in Javascript , so that's why I had to keep this in this file.